.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -58,7 +58,6 @@ body:
       description: Please specify minor version in the above description if relevant.
       multiple: false
       options:
-        - 3.6
         - 3.7
         - 3.8
         - 3.9
```

Comment:
Why do we have 3.9 in here? To catch people who are clearly using the wrong version?

Reply:
No we shouldnt have them! Actually I think we should change these forms, I feel like they are a big barrier of entry for people to report stuff

Reply:
@strickvl @htahir1 Removed it